thrimshim/thrimshim/main.py

```diff
@@ -40,6 +40,9 @@ def metrics():
 	"""Expose Prometheus metrics."""
 	return prometheus_client.generate_latest()
 
+@app.route('/thrimshim')
+def thrimshim_all_rows():
+	return get_all_rows()
```

Comment:
Remove the needless indirection - just put `@app.route` on the `get_all_rows` function itself